pages/common/git-flow.md

```diff
@@ -0,0 +1,24 @@
+# git flow
+
+> A collection of Git extensions to provide high-level repository operations for Vincent Driessen's branching model.
+> More information: <https://github.com/nvie/gitflow>.
+
+- Start using git-flow by initializing it inside an existing git repository:
+
+`git flow init`
+
+- Start developing a new feature. It creates a new branch based on develop:
+
+`git flow feature start <feature>`
+
+- Finish development of a feature, which will merge it into develop, remove the feature branch and switch to develop:
```

Comment:
Could this be explained a bit simpler?

Reply:
Hrm, this is a tricky one to word concisely. Perhaps this will read better?
```suggestion
- Finish development on a feature branch, merging it into the `develop` branch and deleting it:
```